lib/wallaroo/core/recovery/recovery.pony

```diff
@@ -271,6 +273,10 @@ actor Recovery
       promise.next[None](recover this~rollback_complete(_worker_name) end)
       _event_log.initiate_rollback(token, promise)
 
+      let t = Timer(_DelayInitiateRollback( // QQQ HACK
```

Comment:
We need to find the real solution to this problem, since I think this is just masking a bug.

Reply:
Agreed.  I've left the work-around in because it avoids a bug that happens quite frequently.  Also, that bug (or at least a big symptom of it) is #3018.